### ISSUE TITLE
Setup submodule to SteamVR repo

### ISSUE THREAD

**Issue by SimonDarksideJ** [XRTK/OpenVR#2]
# XRTK - Mixed Reality Toolkit Feature Request

## Is your feature request related to a problem? Please describe

Would be good to get this repo started with at least a submodule to the SteamVR repo in the runtime /plugins folder

**Comment by StephenHodgson** [comment]
Is SteamVR different from OpenVR?

**Comment by SimonDarksideJ** [comment]
No :D

OpenVR is basically the Steam API, hence why OpenVR was gone from 2019.3 when Steam wasn't invited :D

**Comment by StephenHodgson** [comment]
Should we rename the repo?

**Comment by SimonDarksideJ** [comment]
No, OpenVR is fine
Or ask Keith and take his advice (since it's his API to maintain)

**Comment by StephenHodgson** [comment]
Yeah I'm also wondering if we should wait until he's finished that UPM package instead of using the submodule

**Comment by SimonDarksideJ** [comment]
I think it's fine to get a headstart on it and work with them on rolling it out.  Our first "Preview" package :D
Plus OpenVR is one of those things EVERYONE asks about (doubt it will be OpenXR anytime soon)
